git-credential-msal

```diff
@@ -159,7 +159,8 @@ def get_http_cache():
     return http_cache
 
 
-def put_http_cache(http_cache):
+def put_http_cache(name: str, http_cache: dict):
+    http_cache_path = os.path.join(xdg.BaseDirectory.xdg_cache_home, name)
```

Comment:
Sorry I didn't notice this before, but this writes outside the cache directory. `name` here is `keyring_name` below, which is the underscorified name whereas the path used by `get_http_cache` looks inside the directory.

Reply:
Oh, oops. My bad. Let me submit a fix for this.

Reply:
#7